Mise au carré mécénat et niveaux + logos

haubourg opened the pull request:
@SebastienPeillet 
@jmarsac 
@mfangain 

Voilà la page que ça donne

![image](https://github.com/OSGeo-fr/QGIS-conf-fr-website/assets/1629853/0a36fac0-b52b-4106-923a-a64e9a939843)

![image](https://github.com/OSGeo-fr/QGIS-conf-fr-website/assets/1629853/a4c19a64-2cf7-4a05-b9a7-ea48a3302e88)

mfangain commented:
Merci Régis!

Quelques petites remarques :
- Partenaires communication : à mettre au singulier s'il n'y a que Géorézo?
- Tarif et visibilité : Note : Cet évènement ne dispose pas d'offre de stand (pour éviter la répétition d'offre)
- Normal que tu affiches 2 fois les montants? (dans le titre et dans le corps du texte)

- Mécènes or : 
** remplacer "citation dans les communiqués de presse" par "citation dans les communications sur les réseaux sociaux"
** espace pour DEPOSER vos flyers (plutôt que présenter, car ça se rapproche un peu trop d'un stand)

- Mécènes argent : 
** le nombre est limité? (il me semblait qu'on avait dit No limit ;-))
** 1 inscription sans S, mais conférenceS avec un S
** espace pour DEPOSER vos flyers (plutôt que présenter, car ça se rapproche un peu trop d'un stand)

- Autre soutien : 
** Serveurs et maintenance associée (sans S)
** le dernier "mais c'est bien quand-même" me gène un peu, car ça sous-entend que c'est mieux de participer aux Journées QGIS que de faire des dons...

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
 Pour toute information ou demande, utilisez le [mail de contact](mailto:qgis-conf@osgeo.asso.fr?subject=[SoutienQGISFR_2024]).
 
-<!-- Plusieurs niveaux sont possibles:
+_Note : Nous n'offrons pas d'offre de stand sur cet évènement_
```
```suggestion
_Note : Note : Cet évènement ne dispose pas d'offre de stand_
```

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
+        <li>Possibilité d'amener un Kakemono sur la scène de la conférence plénière et un autre dans le hall</li>
         <li>Logo, lien et présentation de l'organisme sur le site web</li>
         <li>Logo, lien et courte présentation sur le programme</li>
         <li>Citation dans les communiqués de presse</li>
```
```suggestion
        <li>Citation dans les communications sur les réseaux sociaux</li>
```

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
         <li>Logo, lien et présentation de l'organisme sur le site web</li>
         <li>Logo, lien et courte présentation sur le programme</li>
         <li>Citation dans les communiqués de presse</li>
+        <li>Espace pour présenter vos flyers et goodies dans le hall</li>
```
```suggestion
        <li>Espace pour déposer vos flyers et goodies dans le hall</li>
```

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
 <ul style="padding-left:40px">
-    <li>Montant : 500 € HT</li>
+    <li>Montant : 700 € HT</li>
+    <li>Limité à 5 mécènes</li>
```
```suggestion
```

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
+    <li>Montant : 700 € HT</li>
+    <li>Limité à 5 mécènes</li>
+    <ul style="padding-left:40px">
+        <li>Une inscription incluse dans la journée de conférence</li>
```
```suggestion
        <li>Une inscription incluse dans la journée de conférences</li>
```

SebastienPeillet commented on z30_parrainages.md:
```diff
@@ -94,44 +98,64 @@ Les bénéfices seront dédiés à [soutenir](https://www.qgis.org/fr/site/getin
+        <li>Logo pendant l'évènement (deuxième page d'écran) </li>
+        <li>Logo, lien et présentation de l'organisme sur le site web</li>
+        <li>Logo, lien et courte présentation sur le programme</li>
+        <li>Espace pour présenter vos flyers et goodies dans le hall</li>
```
```suggestion
        <li>Espace pour déposer vos flyers et goodies dans le hall</li>
```

SebastienPeillet reviewed (changes requested):
J'ai ajouté les suggestions de @mfangain qui pouvait être proposé directement. Il y a plus qu'à cliquer dessus.

Il reste les modifs possible sur le partenaire communication, les autres soutiens et la remarque :

> Normal que tu affiches 2 fois les montants? (dans le titre et dans le corps du texte)

perso sur cette dernière, ça ne me choque pas.

haubourg commented:
@mfangain 

> Quelques petites remarques :

> 
>     Partenaires communication : à mettre au singulier s'il n'y a que Géorézo?

c'est un reste de l'an dernier. Pourquoi on avait que Georezo? je trouve ça un peu injuste pour les autres réseaux qui relaient (afiges, CRIGEs). Il y avait un partenariat particulier?
Je commente cette section jusqu'à être clairs sur les partenariats

> 
>     Tarif et visibilité : Note : Cet évènement ne dispose pas d'offre de stand (pour éviter la répétition d'offre)

fixed , merci Seb

> 
>     Normal que tu affiches 2 fois les montants? (dans le titre et dans le corps du texte)
> 

ça vient du template des années précédentes avec le widget qui déplie le détail. Effectivement je simplifie (quitte à perdre le "HT") 

>     Mécènes or :
>     ** remplacer "citation dans les communiqués de presse" par "citation dans les communications sur les réseaux sociaux"

fixed 

>     ** espace pour DEPOSER vos flyers (plutôt que présenter, car ça se rapproche un peu trop d'un stand)

Fixed 

> 
>     Mécènes argent :
>     ** le nombre est limité? (il me semblait qu'on avait dit No limit ;-))

Arg oui, c'est la limite de faire du code pendant une réunion ennuyante, parfois on nous pose des questions. fixed

>     ** 1 inscription sans S, mais conférenceS avec un S
fixed 


>     Autre soutien :
>     ** Serveurs et maintenance associée (sans S)
>     ** le dernier "mais c'est bien quand-même" me gène un peu, car ça sous-entend que c'est mieux de participer aux Journées QGIS que de faire des dons...

oulà ça date de trois ans cette formulation. C'est corrigé en enlevant ce bout de phrase

haubourg commented:
@SebastienPeillet @mfangain c'est bon pour moi, feu flamme?

SebastienPeillet commented:
> @SebastienPeillet @mfangain c'est bon pour moi, feu flamme?

C'est bon pour moi, je te laisse merger. Merci 👍
